lib/contentful/bootstrap/templates/base.rb

```diff
@@ -147,17 +145,37 @@ def create_entries
               end
 
               regular_fields.each do |rf|
-                e[rf.to_sym] = e.delete(rf)
+                value = e.delete(rf)
+                if value.is_a? ::Contentful::Bootstrap::Templates::Links::Base
+                  value = value.to_management_object
+                end
+                e[rf.to_sym] = value
               end
 
-              entry = content_type.entries.create(e.clone)
+              entry = content_type.entries.create({:id => e[:id]})
               entry.save
+
+              e.clone
             end
-          end
+          end.flatten
+
+          entry_list = entry_list.map do |e|
+            puts "Creating Entry #{e[:id]}"
 
-          content_types.each do |content_type|
-            content_type.entries.all.map(&:publish)
+            entry = space.entries.find(e[:id])
+            e.delete(:id)
+            entry.update(e)
+            entry.save
+
+            10.times do |attempt|
+              break if space.entries.find(entry.id).sys[:version] >= 4
```

Comment:
I haven't investigated in detail, but it ends up being 4 when doing those steps with the CMA gem